docs/resources/flow_loglevel.md

```diff
@@ -0,0 +1,56 @@
+---
+page_title: "genesyscloud_flow_loglevel Resource - terraform-provider-genesyscloud"
+subcategory: ""
+description: |-
+  Genesys Cloud flow log level
+---
+# genesyscloud_flow_loglevel (Resource)
+
+Genesys Cloud flow log level
+
+## API Usage
+The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Client has been granted the necessary scopes and permissions to perform these operations:
+
+* todo add links when available
+* [GET /api/v2/flows/instances/settings/loglevels]()
+* [POST /api/v2/flows/{flowId}/instances/settings/loglevels/{flowId}]()
+* [GET /api/v2/flows/{flowId}/instances/settings/loglevels/{flowId}]()
+* [PUT /api/v2/flows/{flowId}/instances/settings/loglevels/{flowId}]()
+* [DELETE /api/v2/flows/{flowId}/instances/settings/loglevels/{flowId}]()
+
+## Example Usage
+
+```terraform
+resource "genesyscloud_flow_loglevel" "flowLogLevel" {
+  flow_id        = "UUID"
+  flow_log_level = "Base"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `flow_id` (String) The flowId for this characteristics set
+- `flow_log_level` (String) The logLevel for this characteristics set
+
+### Read-Only
+
+- `flow_characteristics` (List of Object) Shows what characteristics are enabled for this log level (see [below for nested schema](#nestedatt--flow_characteristics))
```

Comment:
Why is this field listed as a read only attribute? Is this something the user should be able to configure using this resource?

Reply:
these are returned by the api and are configured based on the log level that the user sets, I included them as they're illustrative of the log level after an operation but maybe it's something we don't need?

Reply:
No, it looks like we need them. If you can configure them on a POST request, then they belong here as well. I'm just confused as to why they're listed as read only here. Did you use the `go generate` command for these docs? Nothing should be written manually inside the docs folder because this command will always overwrite it

Reply:
you can include them on a post request but they do not have an impact. The characteristics are set by the `flow_log_level` field even though they are accepted by the post request and returned by it. I haven't edited anything here manually but I'll run `go generate` again